blocks/url-decode/url-decode.js

```diff
@@ -0,0 +1,145 @@
+import { LIBS } from '../../scripts/scripts.js';
+
+const { createTag, getConfig, parseEncodedConfig } = await import(`${LIBS}/utils/utils.js`);
+const { decodeCompressedString } = await import(`${LIBS}/blocks/caas/utils.js`);
+
+const URL_COLUMN = 'caas-url';
+
+/* c8 ignore next */
+const delay = (milliseconds) => new Promise((resolve) => { setTimeout(resolve, milliseconds); });
+
+export const loadQueryIndex = async (url) => {
```

Comment:
Should see if anything like this exists in Milo as a util somewhere, if not you should definitely add it. Great way to batch and get the query index data.

Reply:
Article feed is the closest we have in Milo, I took this from [bulk-updater](https://github.com/adobecom/bulk-updater/blob/main/bulk-update/bulk-update.js#L21-L45).